app/components/post_list/post/index.ts

```diff
@@ -168,6 +168,7 @@ const withPost = withObservables(
             isJumboEmoji,
             isLastReply,
             isPostAddChannelMember,
+            isPostPriorityEnabled: observeIsPostPriorityEnabled(database),
```

Comment:
this seems to be used in the header, so I wonder if it makes sense to move this to the header instead, so when it changes the header of the post is the only one re-rendering instead of the entire post component.

Reply:
We are also using this in the `post.ts` to separate the sequential post to be rendered as a different post in case it is a priority post.